src/index.html

```diff
@@ -9,10 +12,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
It looks like you're trying to link a Sass file directly in your HTML. Browsers do not understand Sass files, so you need to compile it to CSS and link the compiled CSS file instead.